examples/usage.rs

```diff
@@ -0,0 +1,75 @@
+use {
+    anyhow::anyhow,
+    rust_memtester::{MemtestReportList, Memtester, MemtesterArgs},
+    std::{
+        mem::size_of,
+        time::{Duration, Instant},
+    },
+    tracing::{error, info},
+};
+
+fn main() -> anyhow::Result<()> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::DEBUG)
+        .with_writer(std::io::stderr)
+        .init();
+    let start_time = Instant::now();
+    let (mem_usize_count, memtester_args) = match parse_args(std::env::args().skip(1)) {
+        Ok((count, args)) => (count, args),
+        Err(s) => {
+            error!(concat!(
```

Comment:
This is an example where you **don't** want to use `tracing`. This output is actually meant to be read by the user of your program, so you should use `println!()` to send it to `stdout`. Think of `tracing` as stuff that is meant to be read by an engineer that is attempting to figure out why the program is malfunctioning, or perhaps a power user who wants details about all the stuff your program did while it was running. If it's something that the end-user of your program needs to know, print it to `stdout`.